meetings/agenda.md

```diff
@@ -1,14 +1,8 @@
 # Next Technical Steering Committee Meeting Agenda
 
-- Date: 2024-01-08
+- Date: 2024-04-01
```

Comment:
as noted at the very end - it's gonna be Easter Monday and holidays in Europe (and Australia), maybe better postpone